include/boost/geometry/strategies/convex_hull/spherical.hpp

```diff
@@ -0,0 +1,54 @@
+// Boost.Geometry
+
+// Copyright (c) 2020, Oracle and/or its affiliates.
+
+// Contributed and/or modified by Vissarion Fysikopoulos, on behalf of Oracle
+
+// Licensed under the Boost Software License version 1.0.
+// http://www.boost.org/users/license.html
+
+#ifndef BOOST_GEOMETRY_STRATEGIES_CONVEX_HULL_SPHERICAL_HPP
+#define BOOST_GEOMETRY_STRATEGIES_CONVEX_HULL_SPHERICAL_HPP
+
+
+#include <boost/geometry/strategies/convex_hull/services.hpp>
+#include <boost/geometry/strategies/detail.hpp>
+#include <boost/geometry/strategies/spherical/ssf.hpp>
+
+
+namespace boost { namespace geometry
+{
+
+namespace strategies { namespace convex_hull
+{
+
+template
+<
+    typename RadiusTypeOrSphere = double,
+    typename CalculationType = void
+>
+class spherical : strategies::detail::spherical_base<RadiusTypeOrSphere>
```

Comment:
Radius is not needed in this strategy so it could take only `CalculationType`. There is a version `spherical_base` specialized for `void` for that ocasion, i.e.:
```
template <typename CalculationType = void>
class spherical : strategies::detail::spherical_base<void>
```

Reply:
nice, thanks